### ISSUE TITLE
Bitrate of m4a-files is influeced by giving the song a color.

### ISSUE THREAD

**Issue by Eve00000** [mixxxdj/mixxx#12755]
### Bug Description

after giving a m4a song a color, the displayed bitrate changes. Setting color back to black (default) han no effect on the displayed bitrate.
This does not happen on mp3 

### Version

2.5-alpha-229-g7b3e48544a (main)

### OS

win 10

**Comment by ronso0** [comment]
I can not reproduce this (with 2.4).
Could you attach a small screencast demonstrating the issue?

**Comment by Eve00000** [comment]
I had it a couple of times, reordening my history-playlist-tracks in crates. it's in 2.5-alpha-229-g7b3e48544a (main) (not always)
I just tried it a couple of times without reproducing.
This is not a big issue because restarting Mixxx solves it.

**Comment by Swiftb0y** [comment]
I think this is unrelated to the color and something else fishy is happening here. m4a audio files should not have a bitrate > 1000... Also, the recording does not seem to show the rightclick menu probably because of window capture instead of screen capture.

**Comment by Swiftb0y** [comment]
to clarify, did this only happen to tracks where you changed the color? have you tried changing other attributes on these tracks?

**Comment by Eve00000** [comment]
For the record, I have files with a much higher bitrate...
![afbeelding](https://github.com/mixxxdj/mixxx/assets/159239887/bc130a72-0c2e-4781-8496-483f96378291)

As you mentioned, I tried with changing other things (genre ...) and indeed something else 'fishy' is happening, not only with selecting another color. 
Properties -> import metadata again from files solves the problem ..... until I select another file. At that moment the fishy thing happens that changes the bitrate.

**Comment by ronso0** [comment]
Might be related #12467 
Issue was gone after a library rescan.

**Comment by Swiftb0y** [comment]
> For the record, I have files with a much higher bitrate...

Good point, I forgot that m4a files can also contain ALAC which obviously has a much higher bitrate than AAC.

**Comment by Eve00000** [comment]
All my tracks are ALAC, because iTunes does not support FLAC....

**Comment by Eve00000** [comment]
> Might be related #12467 Issue was gone after a library rescan.

after rescan nothing schanged, in id tag has still correct bitrate

**Comment by Swiftb0y** [comment]
so no AAC either than? Is the reading in mixxx correct before you change the color and then wrong or wrong to begin with and only correct after you changed the color, or always wrong (according to your tagger of choice)?

**Comment by Eve00000** [comment]
I don't play compressed music, only lossless 
I tried it on other pc with 2.3.6, no problem: bitrate stays correct after changing color, genre ....

**Comment by Swiftb0y** [comment]
Good to know, what is different about that PC exactly?

**Comment by Eve00000** [comment]
Other pc = version 2.3.6, this is my live-pc

**Comment by Swiftb0y** [comment]
ahh good catch, I wasn't realizing that you were using main. Are you comfortable with compiling code from source? Does the same issue occur with 2.4-beta too?

**Comment by ronso0** [comment]
You can safely test other versions if you backup your data, and roll back if you go back to the stable version.
https://github.com/mixxxdj/mixxx/wiki/Testing

**Comment by Eve00000** [comment]
Thank you, I've read that item. I make a lot of BU's, so no problem of loosing data. It was no big problem testing in a virtual machine.

**Comment by Eve00000** [comment]
2.4.1 : Bitrate is still a problem ....
